iphone/Classes/TiUtils.m

```diff
@@ -201,6 +205,15 @@ +(BOOL)isIOS9_1OrGreater
     return [UITouch instancesRespondToSelector:@selector(altitudeAngle)];
 }
 
++(BOOL)isIOS9_3OrGreater
+{
+#if IS_XCODE_7_3
+    return [WCSession instancesRespondToSelector:@selector(activationState)];
+#else
+    return NO;
+#endif
```

Comment:
Very good question! Checked some more methods from the 9.3 diff, but this one was the only one worked. Will take a look again.